pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1344,27 +1344,19 @@ func TestNodesWherePreemptionMightHelp(t *testing.T) {
 			expected: sets.NewString(),
 		},
 		{
-			name: "ErrReasonAffinityNotMatch should be tried as it indicates that the pod is unschedulable due to inter-pod affinity or anti-affinity",
+			name: "ErrReasonAntiAffinityRulesNotMatch should be tried as it indicates that the pod is unschedulable due to inter-pod anti-affinity",
 			nodesStatuses: framework.NodeToStatusMap{
-				"node1": framework.NewStatus(framework.Unschedulable, interpodaffinity.ErrReasonAffinityNotMatch),
+				"node1": framework.NewStatus(framework.Unschedulable, interpodaffinity.ErrReasonAntiAffinityRulesNotMatch),
 				"node2": framework.NewStatus(framework.UnschedulableAndUnresolvable, nodename.ErrReason),
 				"node3": framework.NewStatus(framework.UnschedulableAndUnresolvable, nodeunschedulable.ErrReasonUnschedulable),
 			},
 			expected: sets.NewString("node1", "node4"),
 		},
 		{
-			name: "pod with both pod affinity and anti-affinity should be tried",
```

Comment:
why was this test removed?

Reply:
the name of the testcase itself is not correct, only the anti-affinity should be tried which has been covered by the testcase above, `ErrReasonAntiAffinityRulesNotMatch should be tried as it indicates that the pod...`